pkg/storaged/jobs.js

```diff
@@ -181,7 +184,16 @@ define([
                 var j = job(path);
 
                 var age_ms = server_now - j.StartTime/1000;
-                return age_ms >= 2000;
+                if (age_ms >= 2000)
+                    return true;
+
+                if (j.ExpectedEndTime > 0) {
+                    var remaining = j.ExpectedEndTime/1000 - server_now;
```

Comment:
We shouldn't declare variables inside of if blocks. Makes the code harder to reason about. Javascript things...